coralproject/talk
# Reject username

## Reject Username
- Adds Reject Username
- Adds a Reject Username Modal

This performs flagging and then rejects the username

## What does this PR do?
- https://www.pivotaltracker.com/story/show/155238146


## How to test this PR?
- Create an Admin
- Create a test user
- Comment with the test user
- Log as admin, and go to the mod queue
- Click the test user
- Reject the username clicking "Actions" next to the name
- A modal should open, Reject the username with a reason
- Change the username from the test user
- Approve or reject from the admin
- Test multiple flows
